GitHub/1SettingUpGit.md

```diff
@@ -1,92 +1,96 @@
-# Set up Git
+# Seting up Git
+
+### Make a GitHub account
+
+First things first, make sure to create a GitHub account if you do not have one already. Head over to https://github.com/join to create one!
 
 ### There are two ways to use git locally on your computer:
 
 - Github CLI
 - Github Desktop
 
-### Installation and setup:
+## Github CLI
 
-#### Github CLI:    
-You can download the executable from the following page for Microsoft Windows. 
-
-Link: https://git-scm.com/download
+### Installation
 
-Once downloaded, follow the on screen prompts to install. As a beginner, it's pretty safe to not change anything and just install the default configuration. Once that is done, you will find a program called *'Git Bash'* in your Start menu. Congratulations! You've successfully installed Git. <br>
+#### Windows
 
-#### For other OSes: 
+You can download the executable from the following page for Microsoft Windows.
 
-**macOS:** 
-Assuming you have Homebrew installed and setup, you can go ahead and run: <br> <br> ``` brew install git ``` <br> <br>
+Link: https://git-scm.com/download
 
-**Ubuntu and derivatives:**
+Once downloaded, follow the on screen prompts to install. As a beginner, it's pretty safe to not change anything and just install the default configuration. Once that is done, you will find a program called _'Git Bash'_ in your Start menu. Congratulations! You've successfully installed Git.
 
-``` sudo apt install git ``` <br> <br>
+#### MacOS
 
-**Fedora, RHEL based distributions:**
+Assuming you have Homebrew installed and setup, you can go ahead and run:
 
-``` sudo dnf install git ``` <br> <br>
+```sh
+brew install git
+```
 
-**SUSE based distributions:**
+#### Linux
 
-``` zypper install git ``` <br> <br>
+There are quite a few distros out there, and you'll need to use their respective package manager.
 
-**Arch based distributions:**
+- **Ubuntu and derivatives:** `sudo apt install git`
 
-``` sudo pacman -S git ``` <br> <br>
+- **Fedora, RHEL based distributions:** ` sudo dnf install git`
 
+- **SUSE based distributions:** `zypper install git`
```

Comment:
Missing sudo here?

Reply:
I didn't change any of the existing commands, but using sudo makes sense